.travis.yml

```diff
@@ -1,13 +1,11 @@
 dist: trusty
 language: python
-isntall:
-  - pip install --user -r requirements.txt
 before_script:
   - ansible --version
   - openstack --version
 script:
   - openstack stack create test -e openstack/test/create.yml -t openstack/ae-environment.yml --enable-rollback --wait --dry-run
-  - ansible-playbook --check -i localhost, ansible/environments.yml
+  - ansible-playbook --check -i localhost, -e 'ansible_python_interpreter="/usr/bin/env python"' ansible/environments.yml
```

Comment:
Was it defaulting to Python3? What was the symptom?

Reply:
It looks like Ansible has [hardcoded python path](https://github.com/ansible/ansible/issues/6345#issuecomment-37107657) to system python, but travis is using virtualenv and [won't find the installed modules](https://travis-ci.org/aeternity/infrastructure/builds/308395778#L990)